internal/pkg/cli/app_package.go

```diff
@@ -179,9 +221,9 @@ func BuildAppPackageCmd() *cobra.Command {
 			return opts.Execute()
 		},
 	}
-	cmd.Flags().StringVarP(&opts.AppName, "name", "n", "", "Name of the application.")
-	cmd.Flags().StringVarP(&opts.EnvName, "env", "e", "", "Name of the environment.")
-	cmd.Flags().StringVar(&opts.Tag, "tag", "", `Optional. The application's image tag. Defaults to your latest git commit's hash.`)
-	cmd.Flags().StringVar(&opts.OutputDir, "output-dir", "", "Optional. Writes the stack template and template configuration to a directory.")
+	cmd.Flags().StringVarP(&opts.AppName, "name", "n", opts.AppName, "Name of the application.")
+	cmd.Flags().StringVarP(&opts.EnvName, "env", "e", opts.EnvName, "Name of the environment.")
+	cmd.Flags().StringVar(&opts.Tag, "tag", opts.Tag, `Optional. The application's image tag. Defaults to your latest git commit's hash.`)
+	cmd.Flags().StringVar(&opts.OutputDir, "output-dir", opts.OutputDir, "Optional. Writes the stack template and template configuration to a directory.")
```

Comment:
Just for clarity would these ever be expected to bind values other than zero-values?

Reply:
Ah the reason why I made this change is that `NewPackageAppOpts` sets the `Tag` value to the commit-id.

If I gave the default value of `""` then it overrides the commitID :(

I'll add a comment explaining this